### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by gituser162**

### CODE REVIEW

**Review by Stepico — APPROVED** [review]
Well done, but it's better to make a decomposition I guess, instead of writing user before each property

File: cypress/e2e/signUp.cy.js [hunk 1]
```diff
@@ -1,5 +1,27 @@
+/// <reference types="cypress" />
+const { generateUser } = require('../support/generate');
```
Comment (murat-yesimbietov) [inline cypress/e2e/signUp.cy.js:2]: Add code of the method to the PR

File: cypress/e2e/signUp.cy.js [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+
+    cy.get('[type="text"]').type(user.username); // хто я?
+
+    cy.get('[type="email"]').type(user.email); // а яка в мене пошта?
+
+    cy.get('[type="password"]').type(user.password); // пароль - Слава Україні
+
+    cy.get('.btn').click(); //тиць на кнопку (назву кнопки не уточнив бо на сайті вона неправильна)
+
+    cy.wait(1000); // більш костильного метода очікування прогрузки сторінки я не знаю
```
Comment (murat-yesimbietov) [inline cypress/e2e/signUp.cy.js:22]: you can add timeout to the assertion method instead of adding cy.wait

**Review by murat-yesimbietov — CHANGES_REQUESTED** [review]
generate.js file is missing in the PR, you should add it

**Review by murat-yesimbietov — APPROVED** [review]

**Pull request closed without merging**